packages/eslint-plugin/src/rules/ngmodule-array-sorted.ts

```diff
@@ -0,0 +1,80 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+import { createESLintRule } from '../utils/create-eslint-rule';
+import { MODULE_CLASS_DECORATOR } from '../utils/selectors';
+import {
+  getDecoratorPropertyValue,
+  isArrayExpression,
+  isIdentifier,
+} from '../utils/utils';
+
+import { Options } from '../utils/property-selector';
+import { NgModule } from '@angular/compiler/src/core';
+
+export const RULE_NAME = 'ngmodule-array-sorted';
```

Comment:
I will rename to module-array-sorted

Reply:
`sort-keys` is a core ESLint rule, so I think it would be more idiomatic to call this rule `sort-SOMETHING`.

As for what `SOMETHING` is, that is subjective, but I'm not a fan of losing the ng off of module, I think that makes it too ambiguous with language level modules.

So my suggestion would be `sort-ngmodule-arrays`

Reply:
Thanks, @JamesHenry I think the biggest struggle with writing my first eslint rule has been what to name it!

Reply:
I went with @JamesHenry suggestion and also decided to treat ngmodule as one word for camel case purposes to keep it consistent. Any further thoughts on that?